lib/features/add_property/view/select_property_type.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:hatspace/features/add_property/view/date_picker_view.dart';
+import 'package:hatspace/features/add_property/view/property_type_cart_view.dart';
+import 'package:hatspace/features/add_property/view_model/cubit/add_property_cubit.dart';
+import 'package:hatspace/features/add_property/view_model/cubit/property_type_cubit.dart';
+
+import 'package:hatspace/strings/l10n.dart';
+import 'package:hatspace/theme/hs_theme.dart';
+
+class SelectPropertyType extends StatelessWidget {
+  const SelectPropertyType({super.key});
+  @override
+  Widget build(BuildContext context) {
+    return BlocConsumer<PropertyTypeCubit, PropertyTypeState>(
+        listener: (context, state) {
+      // Implement validate to enable next button
+      if (state.propertyTypes != null) {
```

Comment:
!= null && isNotEmpty?

Reply:
state.propertTypes is not  a list, therefore isNotEmpty is not appropriate